### PR TITLE
Filter edit and Logout Button

### PR DESCRIPTION
**Pull request by HelenHMaher**
Added Filter Edit screen and made Logout Button functional on Profile Page

### CODE REVIEW

File: src/components/Profile/Profile.css [hunk 1]
```diff
@@ -574,14 +529,21 @@ grid-template-columns: 70px 100px;
...
-.profile__upload_symbol_position 
-input {
-z-index: 5!important;
-visibility: hidden; 
+.profile__upload_symbol_position input {
+  z-index: 5 !important;
+  visibility: hidden;
+}
 
+.profile__filterPage {
```
Comment (HelenHMaher) [inline src/components/Profile/Profile.css:542]: I added this bit

File: src/components/Profile/Profile.css [hunk 2]
```diff
@@ -62,169 +56,156 @@ width: 235px
...
-align-items: center;
-justify-content: center;
-height: 30px;
-width: 80px;
-border-radius: 30px;
+  position: fixed !important;
+  top: 35px;
+  right: 3%;
+  z-index: 2;
+  cursor: pointer;
```
Comment (HelenHMaher) [inline src/components/Profile/Profile.css:199]: I added this bit

**Comment by HelenHMaher** [comment]
I added the logout functionality and the filter menu to the profile page.  It's a bit hard to see the changes because some of the formatting got messed up when I made relatively minor changes.  Let me know if you want me to show you what I did.  I also think this might not be your most recent version of the profile page, but it should be pretty easy to integrate if you want to do that together at some point.

**Review by Filip-Zafran — APPROVED** [review]
A lot what you do is just SiFi to me, but I trust you know WTF you are doing :)

**Pull request merged**